### DISCUSSION TITLE
circular (cyclic) imports for types

### DISCUSSION THREAD

**Discussion by JustFly1984** [oxc-project/oxc#6754] [Q&A] [answered]
Wahat is the benefit to fix circular imports for types? I get why it is bad between packages in workspaces, but why is it bad for the project files? @mattpococ Said that it has no pers issues, so why does oxlint complains on types?

**Comment by Boshen** [comment, marked as answer]
Good questions, we should turn on `ignore_types` by default.

https://github.com/oxc-project/oxc/blob/619d06f00479407ff494acdb72a54d20dffe1f85/crates/oxc_linter/src/rules/import/no_cycle.rs#L26-L27

created an issue: https://github.com/oxc-project/oxc/issues/6759
